packages/joint-core/docs/demo/layout/Port/js/portRotationComp.js

```diff
@@ -56,28 +60,59 @@ paper6.on('cell:pointerclick', function(cellView, e) {
     cellView.model.prop('ports/groups/a/position/args/compensateRotation', !current);
 });
 
-$('<b/>').text('Click on Element to toggle port rotation compensation').appendTo('body');
-$('<br/>').appendTo('body');
+var b1 = document.createElement('b');
+b1.textContent = 'Click on Element to toggle port rotation compensation';
+document.body.appendChild(b1);
+
+var br1 = document.createElement('br');
+document.body.appendChild(br1);
 
-$('<button/>').text('-').appendTo('body').on('click', function() {
+var button1 = document.createElement('button');
+button1.textContent = '-';
+button1.onclick = function() {
```

Comment:
all `onclick()` calls in this file should be changed to `addEventListener()` for consistency